tests/components/freedompro/test_binary_sensor.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for the Freedompro binary sensor."""
+from homeassistant.const import STATE_OFF
+from homeassistant.helpers import entity_registry as er
+
+
+async def test_binary_sensor_get_state(hass, init_integration):
```

Comment:
Would it be more useful to patch the api call, and verify that a change of state propagates through to your integration?

Currently this seems to just initialise the integration then read back the default state (off).